### PR TITLE
Feature/fallback

### PR DESCRIPTION
**Pull request by nivla**
Solve issue #10 fallback methods.

Introduce setFallback/getFallback methods which will be used when the active language does not contain a given language line.

### CODE REVIEW

**Comment by rmariuzzo** [comment]
@nivla I agree with @jhuliano we don't want to have a default fallback. Instead, it will be a normal property (ex. `this.fallback`). The fallback locale will be used only if set and a message could not be found in the specified locale.

Here's an example:

```js
Lang.setMessages(/* 'es' messages only */);

Lang.get('home.welcome');
// > "home.welcome" (the same given key, because no locale and no fallback)

Lang.setLocale('es');
Lang.get('home.welcome');
// > "Bienvenido!"

Lang.setLocale('fr');
Lang.get('home.welcome');
// > "home.welcome" (the same given key, because there's no message for 'home.welcome' in 'fr')

Lang.setFallback('es');
Lang.get('home.welcome');
// > "Bienvenido!" (the message in 'es', because is the fallback)
```

**Comment by nivla** [comment]
It makes sence @jhuliano, thanks @rmariuzzo. I will do the changes and make the PR again.

**Comment by rmariuzzo** [comment]
@nivla you don't need to ‘make the PR again’, just add more commits to this PR. If you need help don't hesitate to ask us.

File: src/lang.js [hunk 1]
```diff
@@ -199,20 +221,22 @@
     Lang.prototype._getMessage = function(key) {
 
         key = this._parseKey(key);
-
+
```
Comment (rmariuzzo) [inline src/lang.js]: @nivla next time, be careful with whitespace added in empty lines.

File: src/lang.js [hunk 2]
```diff
@@ -27,7 +27,7 @@
     // Default options //
 
     var defaults = {
-        defaultLocale: 'en' /** The default locale if not set. */
+        defaultLocale: 'en', /** The default locale if not set. */
```
Comment (rmariuzzo) [inline src/lang.js]: @nivla why?

Comment (nivla) [inline src/lang.js]: Sorry @rmariuzzo i didn't notice that :(. I will be using jshint for future commits.

Comment (rmariuzzo) [inline src/lang.js]: @nivla don't worry too much, I have already created an issue/task to include JsHint (see #21).

Comment (rmariuzzo) [inline src/lang.js]: @nivla don't worry too much, I have already created an issue/task to include JsHint (see #21).

Comment (nivla) [inline src/lang.js]: Good @rmariuzzo, I was testing returning an object with trailing comma in http://jshint.com/ and its pass without not errors, why? that's strange.

**Pull request merged**